proto/authzed/api/v1/core.pb.go

```diff
@@ -212,9 +213,9 @@ type ContextualizedCaveat struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// caveat_name is the name of the caveat expression to use, as defined in the schema *
+	// * caveat_name is the name of the caveat expression to use, as defined in the schema *
```

Comment:
I think we need to fix this in the protos before we merge anything. Go is working on or already supports markdown for doc comments so having our comments randomly turn into bullets is definitely wrong

Reply:
Updated